pyhdb/protocol/lobs.py

```diff
@@ -269,9 +270,9 @@ def _init_io_container(self, init_value):
 
         if PY2 and isinstance(init_value, str):
             # io.String() only accepts unicode values, so do necessary conversion here:
-            init_value = init_value.decode(self.encoding)
+            init_value = cesu8.decode(init_value)
```

Comment:
That modification isn't necessary and isn't the recommended way to decode python strings.